src/app/server/Dnssd.cpp

```diff
@@ -87,26 +86,12 @@ bool DnssdServer::HaveOperationalCredentials()
 void DnssdServer::SetExtendedDiscoveryTimeoutSecs(int32_t secs)
```

Comment:
@chrisdecenzo This method could be called by client applications rather than relying on storage round-trip